app/src/main/java/com/cornellappdev/resell/android/ui/components/global/messages/ChatTag.kt

```diff
@@ -0,0 +1,67 @@
+package com.cornellappdev.resell.android.ui.components.global.messages
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import com.cornellappdev.resell.android.ui.theme.Style
+import com.cornellappdev.resell.android.ui.theme.animateResellBrush
+
+@Composable
+fun ChatTag(
+    text: String,
+    modifier: Modifier = Modifier,
+    active: Boolean = false,
+    venmo: Boolean = false,
+    onClick: () -> Unit,
+) {
```

Comment:
I do have an existing `ResellTag` that somewhat encompasses this. There might be some slight design differences? Is this why you made a different one?